sync-job/base/cron-worfklow.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: sync-job-cw
+spec:
+  schedule: "0 10 14 * *"
+  timezone: "Etc/UTC"
+  concurrencyPolicy: "Replace"
+  startingDeadlineSeconds: 0
+  workflowSpec:
+    paralellism: 4
+    entrypoint: sync-job-wt
+    templates:
+      container:
+        image: quay.io/thoth-station/sync-job:v0.2.3-dev
+        command: ["oc", "process"]
+        args: ["-p IMAGE_VERSION=$IMAGE_VERSION", "-p THOTH_SYNC_FORCE=$THOTH_SYNC_FORCE", "-p THOTH_SYNC_GRACEFUL=$THOTH_SYNC_GRACEFUL", "-p THOTH_SYNC_DEBUG=$THOTH_SYNC_DEBUG", "-f openshift.yaml", " | oc apply -f -"]
+        env:
+          - name: IMAGE_VERSION
+            value: v0.2.3-dev
```

Comment:
This is not needed.
this was used on the sync-job template and is not a part of the application itself. 
https://github.com/thoth-station/sync-job/blob/f630e32b9b62b0e81879a16f95ea34b7e1dadce8/openshift.yaml#L79